prisma/migrations/20230912072258_rename_figma_webhook_table_and_add_client_key_relation/migration.sql

```diff
@@ -0,0 +1,36 @@
+/*
+  Warnings:
+
+  - You are about to drop the `FigmaWebhook` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "FigmaTeamStatus" AS ENUM ('PENDING', 'ACTIVE', 'PAUSED', 'ERROR');
+
+-- DropTable
+DROP TABLE "FigmaWebhook";
+
+-- DropEnum
+DROP TYPE "FigmaWebhookStatus";
+
+-- CreateTable
+CREATE TABLE "FigmaTeam" (
```

Comment:
/nit Same as https://github.com/atlassian-labs/figma-for-jira/pull/58/files#r1323743604